src/coreclr/src/jit/importer.cpp

```diff
@@ -4453,6 +4458,21 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
         }
     }
 #ifdef FEATURE_HW_INTRINSICS
+    else if (strcmp(namespaceName, "System.Numerics") == 0)
+    {
+        CORINFO_SIG_INFO sig;
+        info.compCompHnd->getMethodSig(method, &sig);
+
+        int sizeOfVectorT = 16;
```

Comment:
Why isn't this using `getSIMDVectorRegisterByteLength`?

Reply:
Because I couldn't remember the method name. Will update.

Reply:
Fixed.